### PR TITLE
mcabber 1.1.0

### PR DESCRIPTION
**Pull request by eddiefisher**
- [+] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [+] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [+] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [+] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/mcabber.rb [hunk 1]
```diff
@@ -1,14 +1,8 @@
 class Mcabber < Formula
   desc "Console Jabber client"
   homepage "https://mcabber.com/"
-  url "https://mcabber.com/files/mcabber-1.0.5.tar.bz2"
-  sha256 "a0f200817d2b6196fe4d37918ce16f6fed83a3cef861d7165161e8b1cafcad47"
-
-  bottle do
```
Comment (dunn) [inline Formula/mcabber.rb]: Please leave the bottle block, our CI bot will update that!

File: Formula/mcabber.rb [hunk 2]
```diff
@@ -1,13 +1,10 @@
 class Mcabber < Formula
   desc "Console Jabber client"
   homepage "https://mcabber.com/"
-  url "https://mcabber.com/files/mcabber-1.0.5.tar.bz2"
-  sha256 "a0f200817d2b6196fe4d37918ce16f6fed83a3cef861d7165161e8b1cafcad47"
+  url "https://mcabber.com/files/mcabber-1.1.0.tar.bz2"
+  sha256 "04fc2c22c36da75cf4b761b5deccd074a19836368f38ab9d03c1e5708b41f0bd"
 
   bottle do
-    sha256 "809dca7d8e2f2ed7071db7d27f36632911b0d0f1d7f1c962b081751122e15048" => :sierra
```
Comment (dunn) [inline Formula/mcabber.rb]: Sorry, I meant don't modify the bottle block at all.

Comment (eddiefisher) [inline Formula/mcabber.rb]: ok, I'm moved back bottle block

**Comment by dunn** [comment]
Thanks! Merged as 3c462bbc4eb8eac354a470640da9fa6ae2701b2a.

**Pull request closed without merging**